platforms/chibios/boards/STEMCELL/configs/halconf.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2020 Nick Brassel (tzarc)
+ * Copyright (C) 2022 Mega Mind
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+// #define HAL_USE_DAC TRUE
+#define PAL_USE_WAIT TRUE
+#define PAL_USE_CALLBACKS TRUE
+
+#define HAL_USE_I2C TRUE
+
+#define HAL_USE_SERIAL TRUE
+
+// #define HAL_USE_PWM TRUE
```

Comment:
```suggestion
#define PAL_USE_WAIT TRUE
#define PAL_USE_CALLBACKS TRUE
#define HAL_USE_I2C TRUE
#define HAL_USE_SERIAL TRUE
```
Let's remove the unused ones.